.claude/skills/sveltekit.md

```
@@ -0,0 +1,283 @@
+  {/if}
+</div>
+
+<style>
+  /* scoped by default */
+  p { color: white; }
+</style>
+```
+
+## Route File Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)